2021-09-16 23:16:48,633 INFO    MainThread:40750 [wandb_setup.py:_flush():69] setting env: {}
2021-09-16 23:16:48,633 INFO    MainThread:40750 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-16 23:16:48,633 INFO    MainThread:40750 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20210916_231648-34lsv0lo/logs/debug.log
2021-09-16 23:16:48,633 INFO    MainThread:40750 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20210916_231648-34lsv0lo/logs/debug-internal.log
2021-09-16 23:16:48,634 INFO    MainThread:40750 [wandb_init.py:init():381] calling init triggers
2021-09-16 23:16:48,634 INFO    MainThread:40750 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-09-16 23:16:48,634 INFO    MainThread:40750 [wandb_init.py:init():430] starting backend
2021-09-16 23:16:48,634 INFO    MainThread:40750 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-16 23:16:48,640 INFO    MainThread:40750 [backend.py:ensure_launched():135] starting backend process...
2021-09-16 23:16:48,654 INFO    MainThread:40750 [backend.py:ensure_launched():140] started backend process with pid: 40869
2021-09-16 23:16:48,656 INFO    MainThread:40750 [wandb_init.py:init():435] backend started and connected
2021-09-16 23:16:48,658 INFO    MainThread:40750 [wandb_init.py:init():494] updated telemetry
2021-09-16 23:16:48,659 INFO    MainThread:40750 [wandb_init.py:init():517] communicating current version
2021-09-16 23:16:49,521 INFO    MainThread:40750 [wandb_init.py:init():522] got version response 
2021-09-16 23:16:49,521 INFO    MainThread:40750 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-16 23:16:49,666 INFO    MainThread:40750 [wandb_init.py:init():557] starting run threads in backend
2021-09-16 23:16:54,675 INFO    MainThread:40750 [wandb_run.py:_console_start():1605] atexit reg
2021-09-16 23:16:54,677 INFO    MainThread:40750 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-09-16 23:16:54,677 INFO    MainThread:40750 [wandb_run.py:_redirect():1484] Redirecting console.
2021-09-16 23:16:54,681 INFO    MainThread:40750 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-16 23:16:54,681 INFO    MainThread:40750 [wandb_init.py:init():582] run started, returning control to user process
2021-09-16 23:17:33,796 INFO    MainThread:40750 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 255
2021-09-16 23:17:33,797 INFO    MainThread:40750 [wandb_run.py:_restore():1547] restore
2021-09-16 23:17:36,275 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-16 23:17:36,737 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-16 23:17:36,867 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-16 23:17:37,745 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 12047
}

2021-09-16 23:17:37,848 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 12047
}

2021-09-16 23:17:37,951 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12047
  total_bytes: 12047
}

2021-09-16 23:17:38,054 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12047
  total_bytes: 12047
}

2021-09-16 23:17:38,156 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12047
  total_bytes: 12047
}

2021-09-16 23:17:38,259 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12047
  total_bytes: 12047
}

2021-09-16 23:17:38,484 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12047
  total_bytes: 12047
}

2021-09-16 23:17:38,766 INFO    MainThread:40750 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12047
  total_bytes: 12047
}
local_info {
}

2021-09-16 23:17:40,122 INFO    MainThread:40750 [wandb_run.py:_show_summary():1853] rendering summary
2021-09-16 23:17:40,123 INFO    MainThread:40750 [wandb_run.py:_show_history():1898] rendering history
2021-09-16 23:17:40,124 INFO    MainThread:40750 [wandb_run.py:_show_files():1947] logging synced files
